tests/system/providers/google/cloud/tasks/example_tasks.py

```diff
@@ -51,17 +48,15 @@
 timestamp = timestamp_pb2.Timestamp()
 timestamp.FromDatetime(datetime.now() + timedelta(hours=12))
 
-LOCATION = os.environ.get("GCP_APP_ENGINE_LOCATION", "europe-west2")
+LOCATION = "us-central1"
```

Comment:
we have changed this because the previous version referred directly to "app_engine_http_request"
which was related to the app_engine location, now it is not

this has been changed and adapted to the rest of the test system